migrations/cockroach/V1.7__admin.sql

```diff
@@ -0,0 +1,49 @@
+BEGIN;
```

Comment:
i think this file should be renamed, notification.sql should stay as v1.7 because this is already on master